docs/preview/features/telemetry-enrichment.md

```diff
@@ -36,7 +36,17 @@ ILogger logger = new LoggerConfiguration()
     .Enrich.WithComponentName("My application component")
     .CreateLogger();
 
-logger.Information("This event will be enriched with the application component's name");
+logger.Information("This event will be enriched with the application component's name and by default the environment's machine name");
+```
+
+Or, alternatively one can choose to use the Kubernetes information.
+
+```csharp
+ILogger logger = new LoggerConfiguration()
+    .Enrich.WithComponentName("My application component", RoleInstance.PodName)
```

Comment:
You should not be choosing this here; we should always use the machine name regardless of Kubernetes.
Our application enricher should just extend the machine name one if you ask me.

If they want to include Kubernetes information they should see our Kubernetes enricher next to this one.

Reply:
Bit then the Kubernetes enricher will not overwrite the existing log property bc it's already present. 🤔

Reply:
You don't have to worry about that, the Kubernetes enricher will just provide the info but the logic is up to our `CloudContextConverter`.

https://github.com/arcus-azure/arcus.observability/blob/147bef3db0c6dd5019b1a68a4f6b7861078751cc/src/Arcus.Observability.Telemetry.Serilog.Sinks.ApplicationInsights/Converters/CloudContextConverter.cs#L31

Reply:
Updated this.